lib/migrator.js

```diff
@@ -6,7 +6,7 @@ const { parse: parseUrl } = require('url');
 const { exists, listDir, readFile, writeFile } = require('hexo-fs');
 const parseFeed = require('./feed');
 const { slugize, unescapeHTML } = require('hexo-util');
-const { basename, dirname, extname, join, parse } = require('path');
+const { basename, dirname, extname, join, parse } = require('path').posix;
```

Comment:
```suggestion
const { basename, dirname, extname, join, parse } = require('path').posix;
```

Why `posix`?

Reply:
to use forward slash `/` only, otherwise `join('foo', 'bar')` will results in `foo\bar` in windows.

Reply:
actually I'm also thinking of using `posix` throughout hexo, except for fs operation. [asset_img.js](https://github.com/hexojs/hexo/blob/d96c65b865aaa43dd21c2d1bb1e5e8b0ca49d269/lib/plugins/tag/asset_img.js#L23) currently uses `url.resolve()` (a [legacy](https://nodejs.org/api/url.html#url_url_resolve_from_to) API) to convert backslash to forward slash, the conversion may not be necessary if `posix` is used.

Reply:
If we are handling URL, should we use URL API instead?

Reply:
majority of the functions are dealing with relative path ("/foo/bar/img.jpg") which is similar to posix path; URL API doesn't offer a way to parse folder name and filename.

The only exception is

https://github.com/hexojs/hexo-migrator-wordpress/blob/9cb281149e0181fd42c19280627bca7ddff73c36/lib/migrator.js#L258

`path.join()` is not used there because it will remove the double slash in `http://`.